ci/package.sh

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 
 cd /src
 
-. ./ci/install-build-deps.sh
+. ./ci/install-build-deps.sh $OS
```

Comment:
You don't need to pass it in. If it's defined in the environment (by the caller), use it. Otherwise derive it from `/etc/os-release`.

Reply:
removed passing $OS